config/initializers/assets.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.config.assets.paths << Rails.root.join("vendor", "assets", "javascripts")
+Rails.application.config.assets.paths << Gem.loaded_specs["chosen-rails"].full_gem_path + "/vendor/assets/javascripts"
+Rails.application.config.assets.precompile += %w(chosen-jquery.js)
+Rails.application.config.assets.paths << Rails.root.join("vendor", "assets", "stylesheets")
+Rails.application.config.assets.paths << Gem.loaded_specs["chosen-rails"].full_gem_path + "/vendor/assets/stylesheets"
+Rails.application.config.assets.precompile += %w(chosen.css)
```

Comment:
After the upgrade, Rails couldn't seem to find these `chosen` assets for reasons I could not figure out. This was the only way I was able to get them added correctly. It seems like overkill adding this file just for those assets, but I'm not sure what else to do.